docs/source/app-dev/bindings-scala/index.rst

```diff
@@ -0,0 +1,144 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Scala bindings
+##############
+
+This tutorial provides a basic Scala programmer's introduction to working with Digital Asset distributed ledger,
```

Comment:
I wouldn't call this a tutorial - a tutorial walks you through step by step how to achieve something. I would just say "this page".